cookbook/asset-management.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Asset Management
+hide_table_of_contents: false
+description: Issue a Stellar Asset, deploy it's contract, and mint, burn, freeze, and clawback.
+custom_edit_url: https://github.com/stellar/stellar-cli/edit/main/cookbook/asset-management.mdx
+---
+
+The [stellar-cli] can be used to issue and administer assets on the Stellar network.
+
+The following guide assumes that the [stellar-cli] has been configured to use testnet, or a local test network. To use testnet, run the following command.
+
+```
```

Comment:
looking at the `cmd/crates/soroban-test/tests/it/integration/cookbook.rs` file, seems like all blocks must be `bash` blocks, which means a bunch of other files need updating (I saw a few using `sh`).